annotation/annotation/categories/resources.py

```diff
@@ -153,9 +153,16 @@ def update_category(
     """
     Updates category by id and returns updated category.
     """
-    category_db = update_category_db(
-        db, category_id, query.dict(), x_current_tenant
-    )
+    try:
+        category_db = update_category_db(
+            db, category_id, query.dict(), x_current_tenant
+        )
+    except ValueError as error:
+        raise HTTPException(
+            status_code=400,
+            detail=f"{error}",
+        )
```

Comment:
1. Raise the exception from the original one (from syntax)
2. Use specific error message, do not expose internals to the end user